.github/workflows/integration.yml

```diff
@@ -85,7 +85,11 @@ jobs:
         env:
           ARCHERY_DOCKER_USER: ${{ secrets.DOCKERHUB_USER }}
           ARCHERY_DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_TOKEN }}
-        run: archery docker run -e ARCHERY_INTEGRATION_WITH_RUST=1 conda-integration
+        run: >
+          archery docker run
+            -e ARCHERY_DEFAULT_BRANCH=${{ github.event.repository.default_branch }}
+            -e ARCHERY_INTEGRATION_WITH_RUST=1
+            conda-integration
```

Comment:
```suggestion
          archery docker run \
            -e ARCHERY_DEFAULT_BRANCH=${{ github.event.repository.default_branch }} \
            -e ARCHERY_INTEGRATION_WITH_RUST=1 \
            conda-integration
```

Reply:
@lafiona Could you apply this change to fix CI, and also rebase/merge from git master?

Reply:
Thank you for catching this error! I've committed the change and rebased.